README.md

```diff
@@ -7,6 +7,8 @@ Create [Cycle.js](https://cycle.js.org/) apps with no build configuration.
 * [Getting Started](#getting-started) – How to create a new app.
 * [Custom Flavors](#flavors) – How to develop and publish new flavors
 
+The project is inspired by [create-react-app](https://github.com/facebookincubator/create-react-app)
```

Comment:
I think we have a special mention for this on the Acknowledgements section, I'll rather keep it one place only.